Collections/createVM.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Create a VM
+  hosts: localhost
+  gather_facts: no
+  tasks:
+  - set_fact:
```

Comment:
Please use https://github.com/ansible-collections/community.vmware/tree/main/docs/docsite/rst/vmware_scenarios for creating scenario guides. As per my knowledge, [Remove an existing VMware virtual machine](https://github.com/ansible-collections/community.vmware/blob/main/docs/docsite/rst/vmware_scenarios/scenario_remove_vm.rst#remove-an-existing-vmware-virtual-machine) and [Deploy a virtual machine from a template](https://github.com/ansible-collections/community.vmware/blob/main/docs/docsite/rst/vmware_scenarios/scenario_clone_template.rst#deploy-a-virtual-machine-from-a-template) scenario exists.